### DISCUSSION TITLE
自定义邮件回复模板设置问题

### DISCUSSION THREAD

**Discussion by anghunk** [ArtalkJS/Artalk#329] [Q&A] [answered]
使用官方示例测试：

```html
<div>
    <p>Hi, {{nick}}：</p>
    <p>
        您在 “{{page_title}}” 收到了回复：
    </p>
    <div>
        <div>@{{reply_nick}}:</div>
        <div>{{reply_content}}</div>
    </div>
    <p><a href="{{link_to_reply}}" target="_blank">回复消息 »</a></p>
</div>
```

路径为 `/data/artalk/data/mail_tpl/user.html`

![image](https://user-images.githubusercontent.com/65840178/207278081-456d5f0b-b686-4ab4-ac49-a30715a520b4.png)

重启过容器，测试后邮件如下，一片空白。

![image](https://user-images.githubusercontent.com/65840178/207278282-e61977cc-5b8d-4807-b6d4-6ceec336186c.png)

是我路径设置出错了吗？还是其他问题，大佬康康我的问题！

**Comment by qwqcode** [comment]
执行 `docker-compose logs` 或者 `artalk-go.log` 看看日志有报错吗 🤔

**Comment by scenery** [comment, marked as answer]
应该是没有读取到模版文件，你图中写的相对路径，可能是文件地址错了
```
docker compose exec artalk bash
```

使用上面的命令进容器看看模版文件的路径对不对

如果 docker compose 用的文档中的[默认模版](https://artalk.js.org/guide/deploy.html#docker-compose-%E9%83%A8%E7%BD%B2)，调整下目录结构：
```
- /data
  - /mail_tpl
    - user.html
  - artalk-go.yml
docker-compose.yml
```

然后配置文件中的路径改为：
```
mail_tpl: "/data/mail_tpl/user.html"
```
